test/units/util.sh

```diff
@@ -241,3 +241,111 @@ maybe_umount_usr_overlay() {
         umount -l /usr
     fi
 }
+
+install_extension_images() {
+        local os_release
+        os_release="$(test -e /etc/os-release && echo /etc/os-release || echo /usr/lib/os-release)"
+
+        # Rolling distros like Arch do not set VERSION_ID
+        local version_id=""
+        if grep -q "^VERSION_ID=" "$os_release"; then
+            version_id="$(grep "^VERSION_ID=" "$os_release")"
+        fi
+
+        local initdir="/var/tmp/app0"
+        mkdir -p "$initdir/usr/lib/extension-release.d" "$initdir/usr/lib/systemd/system" "$initdir/opt"
+        grep "^ID=" "$os_release" >"$initdir/usr/lib/extension-release.d/extension-release.app0"
+        echo "${version_id}" >>"$initdir/usr/lib/extension-release.d/extension-release.app0"
```

Comment:
Unnecessary {}.

Reply:
I now realize that most of the issues are pre-existing, but would be nice to fix since you're already touching the code.